pkg/auth0-cli-config-generator/main.go

```diff
@@ -168,7 +180,13 @@ func main() {
 				return err
 			}
 
-			t := tenant{p.clientName, p.clientDomain, token.AccessToken, token.Expiry}
+			t := tenant{
+				Name:        p.clientName,
+				Domain:      p.clientDomain,
+				AccessToken: token.AccessToken,
+				ExpiresAt:   token.Expiry,
+				Scopes:      requiredScopes,
```

Comment:
the CLI is triggering a login flow if the scopes on the tenant are not equal to the defined on the auth package, this is a way we have to "revalidate" tokens if we change the scopes used.

Evidently, it would be hard to maintain (keep in sync) the config generator tenant and login details. I would like to explore that alternative of skip all file-related config if an access token is defined. That way the generator would be repurposed on a command to login and write the access token to a var, no need to know the tenant entity at all.